src/test/groovy/unit/org/pih/warehouse/inventory/CycleCountSpec.groovy

```diff
@@ -265,4 +265,49 @@ class CycleCountSpec extends Specification implements DomainUnitTest<CycleCount>
         expect:
         assert cycleCount.recomputeStatus() == CycleCountStatus.COMPLETED
     }
+
+    void 'getMaxCountIndex should return #maxCountIndex when there are items with #countIndexes count indexes'() {
+        given:
+        Set<CycleCountItem> cycleCountItems = new TreeSet<>()
+        for (int countIndex : countIndexes) {
+            CycleCountItem cycleCountItem = Spy(CycleCountItem)
```

Comment:
Is the Spy and the TreeSet needed because otherwise the items are non-unique because they all have id==null?

If you wanted to avoid needing to mock/stub out the CycleCountItem so that you can test with real methods, you can do:
```
CycleCountItem item = new CycleCountItem(
        countIndex: 1,
        location: bin1,
        status: CycleCountItemStatus.APPROVED
)
item.id = 1
CycleCount cycleCount = CycleCount cycleCount = new CycleCount(
        cycleCountItems: [item]
)
```
Kind of weird to have to set the id manually, but that's the best solution that I've found